setup.py

```diff
@@ -11,7 +11,7 @@
     history = f.read().replace('.. :changelog:', '')
 
 requirements = [
-    'suds-jurko==0.6.0',
+    'suds-community>=1.0.0b1',
```

Comment:
why this version instead of [0.8.5](https://pypi.org/project/suds-community/)

Reply:
https://github.com/suds-community/suds/pull/58#issuecomment-918432499

Reply:
thanks. While when test locally, I found there is some difference between jurko version and the community when build an object. in jurko version, a complex attribute is created regardless its occurrence count or optional or not.

Jurko version
```setattr(data, type.name, value)```

Community version
```setattr(data, type.name, value if not type.optional() or type.multi_occurrence() else None)```

I am not sure if there is customer already depends on this jurko version, if we upgrade to community, this will break those customers. any idea

Reply:
I don't see any issue with this as a customer of Bing. I think this will be fine.

Reply:
This also break our bulk mapping. [Sample code](https://github.com/BingAds/BingAds-Python-SDK/blob/07e72f0aa7ca493815d63f017ac58a12cb1423be/bingads/v13/bulk/entities/audiences/bulk_audience.py#L80) and [here](https://github.com/BingAds/BingAds-Python-SDK/blob/07e72f0aa7ca493815d63f017ac58a12cb1423be/bingads/v13/internal/extensions.py#L1665): the ```entity``` is null with this sudo-community implementation and will bring trouble.

Reply:
Do you have a runnable example or a test case?

Reply:
try this [exmple](https://github.com/BingAds/BingAds-Python-SDK/blob/main/examples/v13/remarketing_lists.py).

Reply:
Please see this issue for more background on why this was changed: https://github.com/suds-community/suds/issues/14, however, that should have reverted the behavior to the 0.6.0 behavior.